docs/getting_started_linux_bazel.md

```diff
@@ -113,11 +113,12 @@ $ ./bazel-bin/iree/tools/iree-run-mlir ./iree/tools/test/simple.mlir -input-valu
 
 ### Further Reading
 
+*   For an introduction to IREE's project structure and developer tools, see the
+    [Developer Overview](./developer_overview.md) guide
```

Comment:
@antiagainst I haven't tested the GitHub pages site with these changes - do you think these relative paths will work if 'developer_overview' isn't nested under 'GetStarted'? I should figure out how to get Jekyll going locally to test...

Reply:
I think it will break. We actually should copy everything to `${BUILD_DIR}/doc/` directory in `build_docs.sh` (instead of copying the "Getting Started" docs under a subdirectory of `doc/GetStarted/`) to be consistent with how these docs are organized in the source code so that the relative paths work as expected. The rendered website structure won't be a problem I think given that's controlled by the `permalink` in the front matter. But need to try out.

Reply:
Permalink didn't seem to be sufficient. I moved the getting started docs under `GetStarted/` in the source tree to match what's on the published website.

Tested links locally with:

1) clone IREE again to `iree-docs/`, checkout `gh-pages` branch there, stay on feature branch in main repo
2) set up Jekyll (I already had Ruby and a bunch of gems installed, had to update the gems with `bundle install` from the `gh-pages` branch in the docs repo) 
3) build docs in main repo:
  ```
  λ bash .\build_tools\cmake\build_docs.sh
  λ python3 ./scripts/prepare_doc_publication.py ./build-docs/doc
  ```
4) copy output into the docs repo:
  ```
  λ cp -rf build-docs/doc/* ../iree-docs/docs/
  ```
5) start local webserver:
  ```
  λ bundle exec jekyll serve
  ```

Reply:
Adding those steps to a README in https://github.com/google/iree/pull/1532